adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -712,9 +749,20 @@ private OrderData createOrderFromPaymentsResponse(final PaymentsResponse payment
         if (paymentsResponse.getAdditionalData() != null) {
             orderData.setAdyenPosReceipt(paymentsResponse.getAdditionalData().get("pos.receipt"));
         }
+
         return orderData;
     }
 
+    private OrderData placePendingOrder(PaymentsResponse.ResultCodeEnum resultCode) throws InvalidCartException {
+        OrderData orderData = getCheckoutFacade().placeOrder();
+
+        OrderModel orderModel = orderRepository.getOrderModel(orderData.getCode());
+        orderModel.setStatus(OrderStatus.ON_HOLD);
```

Comment:
OrderStatus ON_HOLD has a different meaning in order flow. On Hold means -Sourcing Failed. We may need to introduce a different status.

Reply:
Updated to PAYMENT_PENDING